packages/client/lib/sync/beaconsync.ts

```diff
@@ -44,6 +44,17 @@ export class BeaconSynchronizer extends Synchronizer {
     return 'beacon'
   }
 
+  get fetcher():ReverseBlockFetcher|null{
```

Comment:
What's going on with the spacing in here?  Is `prettier` misconfigured or something?

Reply:
hmmm, linter isn't catching/fixing it, added manually